debug/flytolocations.html

```diff
@@ -80,12 +80,12 @@
 
 const trials = [];
 
-map.on('gpu-timing-frame', function(data) {
-    if (measuring) {
+map.on('gpu-timing-deferred-render', function(data) {
+    if (measuring && data.gpuTime !== 0.0) {
         totalGPUTime += data.gpuTime;
         counter += 1;
         const t = totalGPUTime / counter;
-        // console.log(t);
+        console.log(t);
```

Comment:
Leftover?

Reply:
will remove this file anyways in the original PR ;)